src/pages/ReviewedCases.js

```diff
@@ -52,7 +55,7 @@ class ReviewedCases extends Component {
     let data = this.state.electionsList.dul;
     let sortedData = data.sort(function(a, b) {
 
-      if (!a.hasOwnProperty(sortKey) || !b.hasOwnProperty(sortKey)) {
+      if ((isNil(a) || isNil(a[sortKey])) || (isNil(b) || isNil(b[sortKey]))) {
```

Comment:
hasOwnProperty is a function that's used to determine whether or not an object has a property defined on itself. You usually use this in extended classes to determine whether a property was inherited from it's parent class. I have no idea why it's being used here and hasOwnProperty is typically seen as an anti-pattern (now highlighted with the new Babel configuration), hence the adjustment here. Would like some extra eyes for context on this part of the code.